packages/cli/src/services/communityPackages.service.ts

```diff
@@ -250,16 +252,13 @@ export class CommunityPackageService {
 		}
 	}
 
-	async setMissingPackages(
-		loadNodesAndCredentials: LoadNodesAndCredentials,
-		{ reinstallMissingPackages }: { reinstallMissingPackages: boolean },
-	) {
+	async setMissingPackages({ reinstallMissingPackages }: { reinstallMissingPackages: boolean }) {
 		const installedPackages = await this.getAllInstalledPackages();
 		const missingPackages = new Set<{ packageName: string; version: string }>();
 
 		installedPackages.forEach((installedPackage) => {
 			installedPackage.installedNodes.forEach((installedNode) => {
-				if (!loadNodesAndCredentials.known.nodes[installedNode.type]) {
+				if (!this.loadNodesAndCredentials.known.nodes[installedNode.type]) {
```

Comment:
We could have an `isKnown()` method that knows how to deal with nodes and creds. This way we can also keep `known` private.